contracts/RenExSettlement.sol

```diff
@@ -74,28 +77,30 @@ contract RenExSettlement is Ownable {
         Orderbook _orderbookContract,
         RenExTokens _renExTokensContract,
         RenExBalances _renExBalancesContract,
-        uint256 _submissionGasPriceLimit
+        uint256 _submissionGasPriceLimit,
+        address _newSlasherAddress
```

Comment:
Should be `address _slasherAddress` because it is in the constructor, it is not "new". Consistency in naming is important.

Reply:
Fixed 👍